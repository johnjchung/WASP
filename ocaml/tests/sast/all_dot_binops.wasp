Endpoint allBinops(GET:) {
	
	float a = 3 .+ 4;
	float b = 1 .+ 3.5;
	float c = 3.4 .+ 2.4;
	float d = 3.5 .+ 1;
	
	float f = 2 .- 5;
	float g = 4 .- 1.4;
	float h = 2.5 .- 2.2;
	float i = 2.6 .- 1;
	
	float j = 1 .* 5;
	float k = 4 .* 1.3;
	float l = 1.5 .* 14.5;
	float m = 1.6 .* 1;

	float n = 5 ./ 2;
	float o = 10 ./ 2.5;
	float p = 6.6 ./ 3.1;
	float q = 5.5 ./ 2;
}
