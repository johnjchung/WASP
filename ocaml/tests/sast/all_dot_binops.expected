Endpoint allBinops( GET:) {float a = [float] (Binop ([int] 3) .+ ([int] 4));, float b = [float] (Binop ([int] 1) .+ ([float] 3.5));, float c = [float] (Binop ([float] 3.4) .+ ([float] 2.4));, float d = [float] (Binop ([float] 3.5) .+ ([int] 1));, float f = [float] (Binop ([int] 2) .- ([int] 5));, float g = [float] (Binop ([int] 4) .- ([float] 1.4));, float h = [float] (Binop ([float] 2.5) .- ([float] 2.2));, float i = [float] (Binop ([float] 2.6) .- ([int] 1));, float j = [float] (Binop ([int] 1) .* ([int] 5));, float k = [float] (Binop ([int] 4) .* ([float] 1.3));, float l = [float] (Binop ([float] 1.5) .* ([float] 14.5));, float m = [float] (Binop ([float] 1.6) .* ([int] 1));, float n = [float] (Binop ([int] 5) ./ ([int] 2));, float o = [float] (Binop ([int] 10) ./ ([float] 2.5));, float p = [float] (Binop ([float] 6.6) ./ ([float] 3.1));, float q = [float] (Binop ([float] 5.5) ./ ([int] 2));}
